content/events/2020-amsterdam/_index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Amsterdam
+date: '2020-09-10'
+end: '2021-09-11'
+location:
+  country: The Netherlands
+  city: Amsterdam
+  url: https://www.amsterdam.nl/
+  venue: Westergas
+map: amsterdam-map.png
+sched_slug: kubernetescommunitydayams
+social:
+  facebook: https://facebook.com/cloudnativeams
+  twitter: 'https://twitter.com/cloudnativeams'
+sponsors:
+- level: gold
+  orgs:
+  - name: Microsoft
+    logo: microsoft.svg
+    url: https://microsoft.com
+- level: gold
```

Comment:
Please delete this line and the same one on line 26 so that all three gold sponsors will be in the same box.

Reply:
fixed